cumulusci/cumulusci.yml

```diff
@@ -117,18 +117,18 @@ tasks:
     description: Execute a series of REST API requests in a single call
     class_path: cumulusci.tasks.salesforce.composite.CompositeApi
     group: Data Operations
-  connected_app:
+  create_community:
+    description: Creates a Community in the target org using the Connect API
+    class_path: cumulusci.tasks.salesforce.CreateCommunity
+    group: Salesforce Communities
+  create_connected_app:
```

Comment:
Renaming the task config is not backwards compatible; I don't think we should do it even though the name would be clearer.